lib/gollum/views/helpers.rb

```diff
@@ -0,0 +1,35 @@
+require 'yaml'
+
+module Precious
+  module Views
+    module SprocketsHelpers
```

Comment:
This is necessary to be able to use the Sprockets Helpers from within a mustache template. The only way you can pass an argument to a View-method from mustache is for the method to return a `lambda`. So we need to wrap the Helper methods into a `Proc` in order to be able to do this inside a template:
```
{{#stylesheet_tag_mustache}}app{{/stylesheet_tag_mustache}}
```
This yields the same result as `<%= stylesheet_tag 'app' %>` inside an `erb` template. You can also pass in an optional options hash using YAML:
```
{{#stylesheet_tag_mustache}}app media: :all{{/stylesheet_tag_mustache}}
```
This yields the same result as `<%= stylesheet_tag 'app', :media => :all %>` inside an `erb` template.